src/electionguard-ui/ElectionGuard.UI/ViewModels/TallyProcessViewModel.cs

```diff
@@ -75,6 +75,7 @@ public bool IsMultiTally
     private readonly DecryptionShareService _decryptionShareService;
     private readonly ChallengeResponseService _challengeResponseService;
     private readonly MultiTallyService _multiTallyService;
+    private ILogger<TallyProcessViewModel> _logger;

```

Comment:
noted before. Should be `ILogger`, not `ILogger<T>`

Reply:
moved.